_news/2019-02-18-deprecate-1.0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Forseti 1.0 Deprecation Announcement
+author: Henry Chang, Google
+---
+
+We [previously announced the End-of-Life for Forseti 1.0](https://groups.google.com/a/forsetisecurity.org/d/msg/beta-testers/xtQ6dkdRGCI/OSt3-NReBgAJ)
+in October 2018. We have extended that timeline, and this is the last notice
+that Forseti 1.0 will be removed from availability on March 15, 2019.
+
+We strongly encourage any remaining Forseti 1.0 users to migrate to
+Forseti 2.0 as there are many new features to help you to stay secure on
+Google Cloud Platform (GCP).
+
+**New Features**
+
+* New CLI client to operate all Forseti components on demand
+* Parallelized inventory process
+* New inventories available: Bucket IAM Policies, API Configurations,
+Audit Logging Configurations, Networks, Subnetworks, and G Suite Users, and
+[many more](https://forsetisecurity.org/docs/latest/faq/)
```

Comment:
Should link to resource coverage specifically: https://forsetisecurity.org/docs/latest/faq/#google-cloud-platform-resource-coverage